docs/installation/advanced-installation-options.rst

```diff
@@ -41,3 +41,13 @@ with your own certificates:
 
  sudo ./install-prestoadmin.sh <path_to_cert>/cacert.pem
 
+Upgrading Presto-Admin
+----------------------
+Upgrading to a newer version of ``presto-admin`` requires removing the old
+installation and then installing the new version.  To remove the old
+installation, delete the ``presto-admin-install`` and ``/opt/prestoadmin``
```

Comment:
``presto-admin-install`` is inside ``/opt/prestoadmin``, and that's just the recommended directory. Instead, maybe "delete the ``prestoadmin`` directory which contains the ``presto-admin`` script. The recommended installation location is ``/opt/prestoadmin``, but you may have installed it in an alternative location``.